tests/testOpNpyFileReader.py

```diff
@@ -65,6 +70,38 @@ def test_OpNpyFileReader(self):
         finally:
             npyReader.cleanUp()
 
+    def test_OpNpyFileReaderNPZdefault(self):
+        # Now read back our test data using an OpNpyFileReader operator
+        npyReader = OpNpyFileReader(graph=self.graph)
+        try:
+            npyReader.FileName.setValue(self.testDataFilePathNPZdefault)
+
+            # Read the entire file and verify the contents
+            a = npyReader.Output[:].wait()
+            assert a.shape == (10, 11)  # OpNpyReader automatically added a channel axis
+            assert npyReader.Output.meta.dtype == self.testData.dtype
+
+            numpy.testing.assert_almost_equal(a, self.testData)
+        finally:
+            npyReader.cleanUp()
+
+    def test_OpNpyFileReaderNPZnamed(self):
```

Comment:
Thank you for this hint - the reader indeed wasn't behaving in the expected way.